packages/components/inputs/select-utils/src/create-select-styles.ts

```diff
@@ -136,7 +136,10 @@ const controlStyles = (props: TProps) => (base: TBase, state: TState) => {
       if (props.isReadOnly) return 'default';
       return 'pointer';
     })(),
-    padding: `0 ${designTokens.paddingForInput}`,
+    padding:
+      props.appearance === 'quiet'
+        ? `0 ${designTokens.paddingForInputAsQuiet}`
+        : `0 ${designTokens.paddingForInput}`,
```

Comment:
nit:
```suggestion
    padding: `0 ${
      props.appearance === 'quiet'
        ? designTokens.paddingForInputAsQuiet
        : designTokens.paddingForInput
    }`,
```

Reply:
nit: I personally think the current code version is easier to read 📖